diff --git a/Loop/Views/SettingsView.swift b/Loop/Views/SettingsView.swift
index c3ec98b8..01e2d9ae 100644
--- a/Loop/Views/SettingsView.swift
+++ b/Loop/Views/SettingsView.swift
@@ -106,7 +106,7 @@ public struct SettingsView: View {
                 }
             }
             .insetGroupedListStyle()
-            .navigationBarTitle(Text(NSLocalizedString("Settings", comment: "Settings screen title")))
+            .navigationBarTitle(Text(NSLocalizedString("Settings2", comment: "Settings screen title")))
             .navigationBarItems(trailing: dismissButton)
             .actionSheet(item: $actionSheet) { actionSheet in
                 switch actionSheet {
